community/people/gaeg/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: person
+title: Gerson Adimir Esquivel Garcia
+role: Undergraduate Student
+image: Gerson-3 copy.jpg
```

Comment:
Since you changed the name of the file, you need to update it here too

Reply:
Right. Sorry about that. Fixed